ci/install-package.sh

```diff
@@ -78,12 +96,30 @@ echo -e "\e[35m\e[1m Trying to fetch docker image: $IMAGE_NAME:$BRANCH_TAG \e[0m
 if ! docker pull "$IMAGE_NAME:$BRANCH_TAG"
 then
     echo -e "\e[35m\e[1m No worries, we just test against the master branch: $IMAGE_NAME:$MASTER_TAG \e[0m"
-    docker pull $IMAGE_NAME:$MASTER_TAG
-    BRANCH_TAG=master
+    docker pull "$IMAGE_NAME":"$MASTER_TAG"
+    BRANCH_TAG=$MASTER_TAG
+fi
+
+if [ -f ~/.ssh/known_hosts ]
+then
+    MERGE_KNOWN_HOSTS="true"
+    DOCKER_MOUNT_KNOWN_HOSTS_ARGS="--mount type=bind,source=$HOME/.ssh/known_hosts,target=/home/amigo/known_hosts_extra"
 fi
 
 # Run the docker image along with setting new environment variables
-docker run --detach --interactive -e CI=true -e PACKAGE="$PACKAGE" -e BRANCH="$BRANCH" -e COMMIT="$COMMIT" -e PULL_REQUEST="$PULL_REQUEST" --name tue-env "$IMAGE_NAME:$BRANCH_TAG"
+# shellcheck disable=SC2086
+docker run --detach --interactive -e CI="true" -e PACKAGE="$PACKAGE" -e BRANCH="$BRANCH" -e COMMIT="$COMMIT" -e PULL_REQUEST="$PULL_REQUEST" --name tue-env $DOCKER_MOUNT_KNOWN_HOSTS_ARGS "$IMAGE_NAME:$BRANCH_TAG"
+
+if [ $MERGE_KNOWN_HOSTS == "true" ]
+then
+    docker exec tue-env bash -c "sudo chown 1000:1000 ~/known_hosts_extra && ./ssh-merge-known_hosts.py ~/.ssh/known_hosts ~/known_hosts_extra --output ~/.ssh/known_hosts"
```

Comment:
Use the `ssh-merge-known_hosts.py` from tue-env and do not add it into the Dockerfile explicitly.